NEWS.md

```diff
@@ -278,4 +278,4 @@ of syslog-ng, contribute.
 
 We would like to thank the following people for their contribution:
 
-<Fill this by the internal news file creating tool>
+Airbus Commercial Aircraft, Andras Mitzki, Antal Nemes, Attila Szakacs, Balazs Scheidler, Gabor Nagy, Kokan, Laszlo Budai, Laszlo Szemere, László Várady, Mehul Prajapati, Roberto Meléndez, Stephan Marwedel, Thomas De Schampheleire, Vatsal Sisodiya, Vivin Peris
```

Comment:
We should probably add the people who opened issues, we fixed in this release:
Peter Czanik (#3143 -> #3076, ...)
Steven Haigh (#3145 -> #3111)

Reply:
fixed thanks